BeeTracker/app/src/main/java/com/blueprint/foe/beetracker/Model/FactCollection.java

```diff
@@ -0,0 +1,73 @@
+package com.blueprint.foe.beetracker.Model;
+
+import java.util.ArrayList;
+
+/**
+ * Created by johnsington on 2017-07-07.
+ */
+
+public class FactCollection {
```

Comment:
I don't think that this Collection class adds anything to a standard Java collection. I think you can just use an ArrayList of Fact's.